src/SignupRouter.js

```diff
@@ -48,7 +48,7 @@ const Router = () => {
   const isShowKeyboard = store.get && store.get('isMobileKeyboardShown')
   let minHeight = 480
 
-  if (isAndroid && isShowKeyboard) {
+  if (isAndroidWeb && isShowKeyboard) {
```

Comment:
is this tested that its not required on android mobile? this is needed to hide content when keyboard is up
so keyboard doesnt push content up or cover half of the button and it doesnt look good